web/src/styles/operator-shell.css

```
@@ -1300,6 +1513,121 @@
+   each glowing with its speaker's live audio level (--lvl, 0..1). */
+.opr-call-avatars {
+  position: absolute;
+  top: var(--space-3);
+  right: var(--space-3);
+  display: flex;
+  gap: var(--space-2);
+  z-index: 2;
+}
+.opr-call-avatar {
+  --lvl: 0;
+  width: 54px;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)